### PR TITLE
Update service catalog setup.md

### PR DESCRIPTION
**Pull request by crunchypistachiobutter**
add instructions & example for user-defined services

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/tracing/service_catalog/setup.md [hunk 1]
```diff
@@ -21,7 +21,38 @@ further_reading:
 
 ## Overview
 
-Datadog Service Catalog includes all discovered services from APM, USM, and RUM by default. If you are using any of these products, your catalog is pre-populated with entries. 
+Datadog Service Catalog includes all discovered services from APM, USM, and RUM by default. If you are using any of these products, your catalog is pre-populated with entries. If you are not, there are many other ways to add services to your Service Catalog.
+
+## Create User-defined Services
```
Comment (maycmlee) [inline content/en/tracing/service_catalog/setup.md:26]:
```suggestion
## Create user-defined services
```

File: content/en/tracing/service_catalog/setup.md [hunk 2]
```diff
@@ -21,7 +21,38 @@ further_reading:
...
+You can create additional services that are not associated with any Datadog telemetry in Service Catalog. To create such an entry, name your service in the `dd-service` service field in any supported metadata schema versions. For example: 
+
+#### Example
+{{< code-block lang="yaml" filename="service.datadog.yaml" collapsible="true" >}}
+schema-version: v2.1
+dd-service: my-unmonitored-cron-job
+team: shopist
+contacts:
+ - type: slack
+   contact: https://datadogincidents.slack.com/archives/C01EWN6319S
```
Comment (maycmlee) [inline content/en/tracing/service_catalog/setup.md:37]:
```suggestion
   contact: https://datadogincidents.slack.com/archives/XXXXX
```

**Review by maycmlee — COMMENTED** [review]
Just a couple of suggestions. Let me know if this is good to merge.

**Comment by crunchypistachiobutter** [comment]
accepted and good to merge!

**Review by maycmlee — APPROVED** [review]

**Pull request merged**